cluster/saltbase/salt/kube-proxy/kube-proxy.manifest

```diff
@@ -92,6 +92,8 @@ spec:
       readOnly: false
     - mountPath: /run/xtables.lock
       name: iptableslock
+    - mountPath: /lib/modules 
```

Comment:
@cmluciano here are some of the thoughts.

Firstly, we run these containers in privileged mode and probably as a root to be able to manage IPtables and IPVS operations.  So why optimize it as they have already been granted maximum power anyways.

Secondly, its hard to predict it exactly as we wont know which kernel version is currently being used at the host system for instance `ip_vs_wrr` module in our devenv has this many versions of `ipvs_vs_wrr.ko`. 
```
/lib/modules/4.4.0-78-generic/kernel/net/netfilter/ipvs/ip_vs_wrr.ko
/lib/modules/4.4.0-72-generic/kernel/net/netfilter/ipvs/ip_vs_wrr.ko
/lib/modules/4.4.0-75-generic/kernel/net/netfilter/ipvs/ip_vs_wrr.ko
/lib/modules/4.4.0-66-generic/kernel/net/netfilter/ipvs/ip_vs_wrr.ko
/lib/modules/4.4.0-59-generic/kernel/net/netfilter/ipvs/ip_vs_wrr.ko
/lib/modules/4.4.0-70-generic/kernel/net/netfilter/ipvs/ip_vs_wrr.ko
/lib/modules/4.4.0-71-generic/kernel/net/netfilter/ipvs/ip_vs_wrr.ko
/lib/modules/4.4.0-79-generic/kernel/net/netfilter/ipvs/ip_vs_wrr.ko
```
there is no `/lib/modules/latest` or some sort.  Please let us know if there is any standard way to do this.  We may need to mount two modules
`/lib/modules/*/kernel/net/ipv4/netfilter/`  and `/lib/modules/*/kernel/net/netfilter/ipvs`

and Finally, If we want to control the mount to an extent then we may try making it read-only _(if it did not have any side-effects)_ .  We have not tried that yet.  what do you think?